### PR TITLE
Upgrade to Halogen 0.5.0, remove Breadcrumps & Navbar; fix #7

### LINKED ISSUE #7: Upgrade to Halogen 0.5.0

### PR DESCRIPTION
**Pull request by jonsterling**
This commit upgrades this library to Halogen 0.5.0, and removes a couple of wrappers that are no longer compatible with Halogen following discussion with @garyb; these may be rewritten in the future as components, but it makes sense for this library to remain a "barebones" typed interface to the Bootstrap classes, rather than a framework in its own right.

I have also added a self-contained build (`package.json`, `.travis.yml`); the build now generates documentation.

:fire: Do not merge this until this PR has been updated to point to the actual release of Halogen 0.5.0. :fire:

### CODE REVIEW

**Comment by garyb** [comment]
Perhaps we should add a basic README too, even if it's just like this one: https://github.com/slamdata/purescript-halogen-css/blob/master/README.md

**Comment by garyb** [comment]
Actually, maybe this is a better example as it has the bower badge too: https://github.com/garyb/purescript-debug/blob/master/README.md

I've set up a dependency badge for this repo too:

[Dependency Status](https://www.versioneye.com/user/projects/5620ca2f36d0ab0019000841)

**Comment by jonsterling** [comment]
@garyb Good idea! I've added a README w/ the badges.

File: bower.json [hunk 1]
```diff
@@ -11,8 +10,8 @@
     "output"
   ],
   "dependencies": {
-      "purescript-halogen": "30e8b2c7174a1eeda73f67cc42c739ca24a1e218",
-      "purescript-prelude": "^0.1.1"
+      "purescript-halogen": "master",
```
Comment (garyb) [inline bower.json]: v0.5.0? :)

Comment (jonsterling) [inline bower.json]: Heh, I knew I'd forgotten one!

Comment (jonsterling) [inline bower.json]: ok, fixed now :smile:

**Comment by garyb** [comment]
:+1:

**Pull request merged**